app/views/subscriptions/new.html.haml

```diff
@@ -12,33 +14,35 @@
 
       = f.hidden_field :search_criteria
 
-      .govuk-inset-text
+      .grouped-text-block
         %ul.govuk-list
           - @subscription.filtered_search_criteria.each_pair do |filter, value|
             %li
               - if filter.present?
                 %span{ class: 'govuk-!-font-weight-bold' } #{filter.humanize}:
               = value
 
-      %p.govuk-body-l=t('subscriptions.next_steps')
-
       = f.govuk_email_field :email,
         label: { text: t('subscriptions.labels.email_html'), size: 's' },
         hint_text: t('subscriptions.hints.email'),
         required: true
 
-      = f.govuk_text_field :reference,
-        label: { text: t('subscriptions.labels.reference_html'), size: 's' },
-        hint_text: t('subscriptions.hints.reference'),
-        required: true
+      = f.govuk_collection_radio_buttons :frequency,
+        [['daily', t('subscriptions.frequency.daily')], ['weekly', t('subscriptions.frequency.weekly')]],
```

Comment:
These options should be defined somewhere else probably, maybe on the subscription model as `FREQUENCY_OPTIONS`